build.sbt

```diff
@@ -396,11 +387,11 @@ lazy val publishBridgesAndSet = {
     val userScalaVersion = args.mkString("")
     s"${compilerInterface.id}/publishLocal" ::
       compilerBridgeScalaVersions.flatMap { (bridgeVersion: String) =>
-      s"wow $bridgeVersion" ::
+      s"++ $bridgeVersion!" ::
```

Comment:
The good thing about this is that we won't have OOM errors with `++` anymore in sbt 1.0.